test/security/unsigned_subpackets.js

```diff
@@ -81,7 +81,7 @@ async function makeKeyValid() {
   pusersig.readSubPackets(fake.writeHashedSubPackets(), false);
   // reconstruct the modified key
   const newlist = new PacketList();
-  newlist.concat([pubkey, puser, pusersig]);
+  newlist.push(...[pubkey, puser, pusersig]);
```

Comment:
This is then very nitpicky but you can just do:
```suggestion
  newlist.push(pubkey, puser, pusersig);
```

Reply:
Yes this is fixed in the next PR :upside_down_face: